ui-library-quickstart-teams-interop-meeting-chat/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "ui-library-quickstart-teams-interop-meeting-chat",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@azure/communication-react": "1.8.0-beta.1",
+    "@fluentui/react": "8.101.0",
+    "@testing-library/jest-dom": "^5.14.1",
```

Comment:
Do we want to include the testing dependencies?

Reply:
good catch